pkg/must-gather/mcp/mcp.go

```diff
@@ -0,0 +1,199 @@
+package mcp
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/modelcontextprotocol/go-sdk/mcp"
+	omcclient "github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/must-gather/omc-client"
+	ovsdbtool "github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/must-gather/ovsdb-tool"
+)
+
+// MustGatherMCPServer is a server for the must gather MCP.
+type MustGatherMCPServer struct {
+	OmcClient *omcclient.OmcClient
+	OvsdbTool *ovsdbtool.OvsdbTool
+}
+
+// NewMCPServer creates a new MustGatherMCPServer. It will return an error if the omc client
+// cannot be created.
+func NewMCPServer() (*MustGatherMCPServer, bool) {
+	omcClient, err := omcclient.NewOmcClient()
+	if err != nil {
+		log.Printf("Failed to create omc client: %v", err)
+		return nil, false
```

Comment:
Nit: bool return may not be required since you anyway return nil here, it can be used at the caller side.